src/main/kotlin/org/cascadebot/cascadebot/events/GeneralEventListener.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.events
+
+import net.dv8tion.jda.api.JDA
+import net.dv8tion.jda.api.events.ReadyEvent
+import net.dv8tion.jda.api.events.StatusChangeEvent
+import net.dv8tion.jda.api.events.guild.GuildJoinEvent
+import net.dv8tion.jda.api.events.guild.GuildLeaveEvent
+import net.dv8tion.jda.api.events.guild.member.GuildMemberJoinEvent
+import net.dv8tion.jda.api.events.guild.member.GuildMemberLeaveEvent
+import net.dv8tion.jda.api.events.role.RoleDeleteEvent
+import net.dv8tion.jda.api.hooks.ListenerAdapter
+import net.dv8tion.jda.api.requests.ErrorResponse
+import org.apache.commons.lang3.StringUtils
+import org.cascadebot.cascadebot.CascadeBot
+import org.cascadebot.cascadebot.UnicodeConstants
+import org.cascadebot.cascadebot.data.Config
+import org.cascadebot.cascadebot.data.managers.GuildDataManager
+import org.cascadebot.cascadebot.messaging.MessageType
+import org.cascadebot.cascadebot.utils.DiscordUtils
+import org.cascadebot.cascadebot.utils.FormatUtils
+import org.cascadebot.cascadebot.utils.placeholders.PlaceholderObjects
+import java.util.function.Consumer
+
+class GeneralEventListener : ListenerAdapter() {
```

Comment:
So you combined some, but not all of the event listeners?

Reply:
What do you mean? I've not combined any listeners? 

I converted the general event listener to Kotlin and added to it

Reply:
If this isn't a combined listener, then your new stuff should be its own listener as convention through out the rest of the code.

Reply:
why u do dis